index.html

```diff
@@ -261,7 +311,7 @@ <h4 class="media-text">LET’S CONNECT</h4>
                                 <li><a href="#" class="skill-link">Skills</a></li>
                             </ul>
                         </div>
-                        <div class="v-icon">
+                        <div class="v-icon rotate">
```

Comment:
- [ ] Kindly make sure you stick to the original design provided in Figma. the `typo:(font-size)`, the `width` and `height` of the icons must match.
As you can see your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193147038-98822b4b-df03-42fa-9634-dd34e189db5b.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193148040-6241e2f0-b945-49b2-ad7b-7337cd4eec19.png)